docs/sources/flow/reference/components/prometheus.exporter.blackbox.md

```diff
@@ -33,7 +33,8 @@ Omitted fields take their default values.
 
 Name | Type | Description | Default | Required
 ---- | ---- | ----------- | ------- | --------
-`config_file`                 | `string`       | Blackbox configuration file with custom modules. | | yes
+`config_file`                 | `string`       | Blackbox configuration file with custom modules. | | no
+`config`                      | `string`       | Blackbox configuration with custom modules as YAML. | |no
 `probe_timeout_offset`        | `duration`     | Offset in seconds to subtract from timeout when probing targets.  | `"0.5s"` | no
 
 The `config_file` argument points to a YAML file defining which blackbox_exporter modules to use. See [blackbox_exporter]( https://github.com/prometheus/blackbox_exporter/blob/master/example.yml) for details on how to generate a config file.
```

Comment:
Could we clarify that `config` is using inline yaml (or whatever is called, preferably with a link)?

Reply:
I hope its more clear now!